pom.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-spring-boot-parent</artifactId>
+        <version>1.5.0-rc1</version> <!-- lookup parent from repository -->
+    </parent>
+    <artifactId>overseas-entities-api</artifactId>
+    <version>unversioned</version>
+    <name>overseas-entities-api</name>
+    <description>API for register an overseas entity service</description>
+    <properties>
+        <java.version>11</java.version>
```

Comment:
I though we are on `1.8`?

Reply:
We should be safe to use 11, had a quick look and insolvency-data-api is also using 11 - https://github.com/companieshouse/insolvency-data-api/blob/main/pom.xml#L19
Also our confirmation-statement-api is running  11 - https://github.com/companieshouse/confirmation-statement-api/blob/main/pom.xml#L15

Reply:
Good,I will change the version on my local profile config to work with the 11